pkg/helper/helpers.go

```diff
@@ -62,6 +62,13 @@ func RankMapStringInt(values map[string]int) []string {
 	for i, kv := range ss {
 		ranked[i] = kv.Key
 	}
+	sort.Slice(ranked, func(i, j int) bool {
```

Comment:
To be honest I don't fully understand the `RankMapStringInt` function.
Could you please add a small example to the function description? or some explanation.

From my understanding, we are trying to find the most common prefix.
Would a `findMax` function will work here? to find the entry in the map with the maximal value?

Reply:
I think this should help understand https://github.com/openshift/rosa/pull/935/files#diff-fb37be3d2646b1c87d4ba50f69ca7ee651bb3c4a0dec17820c622915d8975241R32

Finding a single entry with maximum value requires sorting to keep consistency of which element we will retrieve in case there are two entries with the same maximal value

Reply:
Thanks for the explanation.
I would recommend adding some comments to **RankMapStringInt** or breaking it into smaller functions as we discussed.